lib/Zonemaster/Backend/DB/PostgreSQL.pm

```diff
@@ -81,7 +81,7 @@ sub create_schema {
     ) or die Zonemaster::Backend::Error::Internal->new( reason => "PostgreSQL error, could not create 'test_results' table", data => $dbh->errstr() );
 
     $dbh->do(
-        'CREATE INDEX IF NOT EXISTS test_results__hash_id ON test_results (hash_id)'
```

Comment:
Don't we still need an index for just hash_id? E.g. for the `test_progress` RPCAPI method.